### PR TITLE
[Gravatar Proxy] Add functional test

### PR DESCRIPTION
**Pull request by loic-sharma**
Test run: https://nuget.visualstudio.com/NuGetBuild/_build/results?buildId=63423
Test run 2: https://nuget.visualstudio.com/NuGetBuild/_build/results?buildId=63525

Addresses https://github.com/NuGet/NuGetGallery/issues/7588

### CODE REVIEW

File: tests/NuGetGallery.FunctionalTests.Core/Helpers/UrlHelper.cs [hunk 1]
```diff
@@ -198,6 +198,11 @@ public static string GetContactOwnerPageUrl(string packageId)
             return BaseUrl + @"Packages/" + packageId + "/ContactOwners";
         }
 
+        public static string GetAvatarUrl(string accountName)
+        {
+            return BaseUrl + $@"profiles/{accountName}/avatar";
```
Comment (joelverhagen) [inline tests/NuGetGallery.FunctionalTests.Core/Helpers/UrlHelper.cs]: should only need `$` right?

File: tests/NuGetGallery.FunctionalTests/ProfileTests.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        [Priority(1)]
+        [Category("P1Tests")]
+        public async Task GetProfileAvatarAsync()
+        {
+            // Act
+            using (var httpClient = new HttpClient())
+            using (var response = await httpClient.GetAsync(UrlHelper.GetAvatarUrl("microsoft")))
+            {
+                // Assert
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
```
Comment (joelverhagen) [inline tests/NuGetGallery.FunctionalTests/ProfileTests.cs]: nit: perhaps check the content is non-empty

File: tests/NuGetGallery.FunctionalTests.Core/Helpers/UrlHelper.cs (hunk 1 above)
Comment (loic-sharma) [reply tests/NuGetGallery.FunctionalTests.Core/Helpers/UrlHelper.cs]: Yup. Copy paste error :)

File: tests/NuGetGallery.FunctionalTests/ProfileTests.cs (hunk 2 above)
Comment (loic-sharma) [reply tests/NuGetGallery.FunctionalTests/ProfileTests.cs]: Added

**Pull request merged**